packages/slice-machine/lib/models/common/widgets/Image/components.tsx

```diff
@@ -75,84 +78,103 @@ export const AddThumbnailButton = ({ onClick }) => (
-        helpers.setValue({ ...field.value, [key]: value });
-      };
+    (key: string, fn = (v: string): number | string => v) =>
+    (e: React.ChangeEvent<HTMLInputElement>) => {
+      helpers.setTouched(true);
+      const cast = fn(e.target.value);
+      const value =
+        e.target.type === "number"
+          ? typeof cast === "number" && isNaN(cast)
+            ? null
+            : cast
+          : cast;
+      helpers.setValue({ ...field.value, [key]: value });
+    };
 
   return (
     <Fragment>
       <FormFieldInput
-        // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/restrict-template-expressions
         fieldName={`${prefix}.name`}
         meta={{
           ...meta,
-          // eslint-disable-next-line @typescript-eslint/no-unsafe-return, @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-assignment
-          error: meta.error && meta.error.name,
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore:disable
+          // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+          error: meta.error && meta.error?.name,
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore:enable
         }}
         formField={{ label: `Name${requiredChar}`, placeholder: "main" }}
         field={
           prefix === "config.constraint"
-            ? { value: "main", readOnly: true }
-            : // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/restrict-template-expressions, @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
+            ? // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+              // @ts-ignore:disable
+              { value: "main", readOnly: true }
+            : // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
               { value: field.value.name, onChange: createSetField("name") }
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore:enable
         }
         variant={prefix === "config.constraint" ? "disabled" : "primary"}
         sx={{ mb: 3 }}
       />
       <FormFieldInput
-        // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
         fieldName={`${prefix}.width`}
         meta={{
           ...meta,
-          // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/restrict-template-expressions, @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-assignment
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore:disable
+          // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
           error: meta.error && meta.error.width,
```

Comment:
I tested the form and it seems `error` is always a string and it actually never displayed! If you need to know where the error comes from (name, width or height) you will need additional logic/checks

Reply:
I manged to get an object but i had to change the type of the field so the validator would fail :/